src/main/java/nextstep/subway/line/domain/Sections.java

```diff
@@ -65,6 +65,10 @@ private void updateDownStationIfSameDownStation(Section section) {
                 .ifPresent(it -> it.updateDownStation(section.getUpStation(), section.getDistance()));
     }
 
+    public List<Section> getSections() {
+        return sections;
```

Comment:
외부에 sections의 참조가 노출되면 addSection의 호출 없이 section을 추가할 수 있어, 데이터 정합성이 깨질 우려가 있어 보이네요~

```suggestion
        return Collections.unmodifiableList(sections);
```

Reply:
그냥 컬렉션을 그대로 내려주면 외부에서 데이터 조작이 가능할수도 있겠네요
이부분 Collections. unmodifiableList로 한번 감싸서 반환하도록하였습니다.